.github/pull_request_template.md

```diff
@@ -4,8 +4,12 @@ Include a summary of the changes and the related issue.
 
 __Related to:__ `(ClickUp/JIRA task name)`
 
+__Any expected test failures?__
```

Comment:
How do you suppose we use this?

Reply:
For example, if we know that some tests might fail because of incompatibility with python3.11, we can mention that here. Also, any failures due to known issues can be mentioned here

Reply:
Ok.
We know of the dependency issues between the newer and older versions of Python.
Besides that, how often do tests fail due to something else?

Reply:
Happens occasionally.
In any case, better to mention it outright here so that the reviewer doesn't spend time trying to debug or enquire about failing tests